.github/pull_request_template.md

```diff
@@ -31,13 +33,15 @@ Use a check for ✓ the following type of changes:
 
 ## Testing Steps / QA Criteria
 
+N/A
+
 [Describe the steps taken to test upcoming changes. Describe any additional testing that needs to be done beyond the steps outlined above. This might include manual testing, exploratory testing, or other types of testing that can't be easily automated.]
 
 ## Checklist:
 
 -   [ ] This PR is up to date with the development branch, and merge conflicts have been resolved
--   [ ] My code follows the style guidelines of this project
+-   [✓ ] My code follows the style guidelines of this project
 -   [ ] I have executed npm run lint and resolved any outstanding errors. 
--   [ ] I have performed a self-review of my own code
+-   [✓ ] I have performed a self-review of my own code
 -   [ ] I have commented my code, particularly in hard-to-understand areas
--   [ ] My changes generate no new warnings
+-   [✓ ] My changes generate no new warnings
```

Comment:
Michelle, pull request template is to be kept blank for future use. The one to be edited will autopopulate in  github. Can you revert this back to the original and then submit another commit?